tools/integrations/manage.sh

```diff
@@ -69,6 +89,34 @@ cmd_publish() {
   echo "Publishing new version ($versioned_image)"
   docker push "$versioned_image"
   docker push "$latest_image"
+
+  if [[ "true" == "${publish_spec_to_cache}" ]]; then
```

Comment:
Should there be extra conditions here to exclude normalization docker images when publishing? see https://github.com/airbytehq/airbyte/pull/6052#issuecomment-925120639